nrf_security/cmake/symbol_rename.cmake

```diff
@@ -12,50 +12,108 @@
 # If two or more backends are enabled for a given function, each of them will
 # be renamed to be unique and a glue layer will be used as new front-end.
 #
+
+#
+# This macro comments out the instruction to rename a symbol
+# in case the configuration is not glued and enabled in backend
+#
 macro(keep_config_test_glue mbedtls_config backend_name)
   if(NOT CONFIG_GLUE_${mbedtls_config}
      AND CONFIG_${backend_name}_${mbedtls_config})
     set(KEEP_${mbedtls_config} "#")
   endif()
 endmacro()
 
+#
+# This macro comments out the instruction to strip a symbol
+# if the configuration is not enabled in the backend
```

Comment:
Would it be valid to say

>  if the configuration is not enabled in the backend

?

Or is this trying to say something else?

Reply:
Fixed with slightly different text